### PR TITLE
Fix typo in name of CliffordPushThroughMeasures pass

### PR DESCRIPTION
**Pull request by isobelhooper**
# Description

This PR fixes a typo in the JSON `name` field for the CliffordPushThroughMeasures pass. The pass is correctly named in the Python bindings, but the typo shows up in the output of `.to_dict()` for the generated compiler pass.

This pass name isn't used in any tests (or output in the API docs, I think), so no test or docs changes are needed.

# Checklist

- [x] I have performed a self-review of my code.
- [ ] I have commented hard-to-understand parts of my code.
- [ ] I have made corresponding changes to the public API documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have updated the changelog with any user-facing changes.

### CODE REVIEW

**Review by sjdilkes — DISMISSED** [review]

**Comment by cqc-alec** [comment]
Hi @isobelhooper , when you make any change to the tket source you have to bump the tket patch version number in `tket/conanfile.py` and correspondingly the required version in `pytket/conanfile.py`. This is why the CI is failing.

**Comment by isobelhooper** [comment]
> Hi @isobelhooper , when you make any change to the tket source you have to bump the tket patch version number in `tket/conanfile.py` and correspondingly the required version in `pytket/conanfile.py`. This is why the CI is failing.

Thank you for this - have now made those changes in the commit as well!

**Comment by cqc-alec** [comment]
Ah, looks like homebrew have removed clang-format 18 so the formatting check is failing. I'll fix this in a separate PR...

File: pytket/conanfile.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def package(self):
         cmake.install()
 
     def requirements(self):
-        self.requires("tket/1.3.24@tket/stable")
+        self.requires("tket/1.3.25@tket/stable")
```
Comment (cqc-alec) [inline pytket/conanfile.py:35]:
```suggestion
        self.requires("tket/1.3.26@tket/stable")
```

File: tket/conanfile.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 class TketConan(ConanFile):
     name = "tket"
-    version = "1.3.24"
+    version = "1.3.25"
```
Comment (cqc-alec) [inline tket/conanfile.py:26]:
```suggestion
    version = "1.3.26"
```

**Review by cqc-alec — CHANGES_REQUESTED** [review]
Version needs bumping again.

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**